extensions/wikia/EmbeddableDiscussions/EmbeddableDiscussions.i18n.php

```diff
@@ -35,6 +37,7 @@
 	'embeddable-discussions-zero' => 'Es gibt noch keine Beiträge.',
 	'embeddable-discussions-zero-detail' => 'Erstelle einen Beitrag und starte eine neue Diskussion!',
 	'embeddable-discussions-forum-name' => 'In $1',
+	'embeddable-discussions-share-heading' => 'Beitrag teilen',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "embeddable-discussions-share-heading" 10 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)